ast.c

```diff
@@ -884,6 +921,16 @@ static void ast_to_zval(zval *zv, zend_ast *ast, ast_state_info_t *state) {
 		ast_update_property_long(&prop_group_zval, AST_STR(str_flags), 0, AST_CACHE_SLOT_FLAGS);
 	}
 #endif
+#if PHP_VERSION_ID < 80000
+	if (ast->kind == ZEND_AST_CLASS_CONST_DECL && state->version >= 80) {
+		zval const_decl_zval = *zv;
+		zval attributes_zval = *zv;
```

Comment:
The initialization to zv doesn't make sense here.

Reply:
That was a mistake, copied from above line without removing initialization